### DISCUSSION TITLE
v1  - UX 🍿

### DISCUSSION THREAD

**Discussion by alisonjoseph** [carbon-design-system/carbon-platform#1272] [Release feedback]
**Hi 👋  Thanks for joining our v1 "Test Fest"!** 

Below are some templates for feedback on the topic of fixes, visuals, navigation, and requests. We'd greatly appreciate if you provide suggestions to solve any issues you see. ❤️ 

```
📝 Fix
**Page:** URL
**Issue:** E.g. Change the large break point from 9 to 8 columns. 
```

```
🎨 Visual
**Page:** URL
**Issue:** E.g. Icons on this page have low contrast, which isn't accessible. 
**Suggestion:** E.g. Change color from x to y. 
```

```
🧭 Navigation and IA feedback
**Page:** URL
**Issue:** Example
**Suggestion:** Example
```

```
🙏 Request
**Page:** URL
**Issue:** E.g. I'm struggling to understand what status is in the filters and in the list of library assets
**Suggestion:** E.g. Maybe there could be a tooltip or link to a glossary in the schema.
```

**Comment by kingtraceyj** [comment]
🧭 Navigation and IA feedback
**Page:** [library navigation](https://next.carbondesignsystem.com/libraries/carbon-charts/latest/assets)
**Issue:** If a user come to a library from another place other than the library catalog, the "Back to all libraries" doesn't make sense. 
![image](https://user-images.githubusercontent.com/20582179/191102504-9a2019c2-794a-45fc-9624-1d0edfa61a58.png)

**Suggestion:** Change wording to All libraries rather than back

**Reply by mattrosno** [reply]
We aren't using the word "All" in other navigational items (library and asset detail dashboard buttons)... so just "Libraries" makes sense to me.

**Reply by kingtraceyj** [reply]
#1324

**Comment by aubrey-oneal** [comment]
🎨 Visual: Re-work design kit dependency image
**Page:** [Designing Get Started](https://next.carbondesignsystem.com/designing/get-started#step-2:-access-design-tools)
**Issue:** We let the dependency image go to the backlog. We need to revisit it so that it matches the ecosystem language well. When I download the image it is "webp" file type which is odd. 
**Suggestion:** This is really hard. I don't have a suggestion yet.

**Reply by kingtraceyj** [reply]
#1325

**Comment by ariellalgilmore** [comment]
🧭 Navigation and IA feedback
**Page:** https://next.carbondesignsystem.com/libraries/ibmdotcom-web-components/latest/assets/callout-quote
**Issue:** When going to an asset page I expected the breadcrumb on the top left to show me only the Carbon for IBM.com components, but instead it brings you back to the component page with no filter
**Suggestion:** Possibly automatically add the filter for the library being used depending on which asset page you are on
<img width="1431" alt="Screen Shot 2022-09-19 at 1 11 37 PM" src="https://user-images.githubusercontent.com/20210594/191107768-113e37b6-5a96-411b-9969-3a2ecbdba41b.png">

**Reply by kingtraceyj** [reply]
@ariellalgilmore thanks for logging this. did you come from the component catalog or from a specific library?

**Reply by ariellalgilmore** [reply]
I came from the dotcom web component library

**Reply by ariellalgilmore** [reply]
@kingtraceyj I came from the dotcom web component library

**Reply by kingtraceyj** [reply]
#1329

**Comment by kingtraceyj** [comment]
🎨 Visual - Line between filter and search
**Page:** [Functions catalog](https://next.carbondesignsystem.com/assets/functions)
**Issue:** There is no line between the filter dropdown and search field 
**Suggestion:** Add `$border-subtle` line in between filter dropdown and search field
![image](https://user-images.githubusercontent.com/20582179/191108561-be2ffc0c-ecaa-4f2f-a105-6bb492c0a680.png)

**Reply by alisonjoseph** [reply]
https://github.com/carbon-design-system/carbon-platform/issues/1323

**Comment by laurenmrice** [comment]
🎨 Visual
**Page:** https://next.carbondesignsystem.com/designing/get-started
**Issue:** Resource card usage — When to use regular size versus small size?
**Suggestion:** I was wondering if there are currently any rules around when to use each size of the Resource card component. On the `Designing > Get started` page, Step 2 (Access design tools) and Step 3 (tutorials) use different size resource cards. Was not sure if this was intentional or not. 

Looks like that maybe we are using the regular size resource card for the most important cards on the page (Design tools) and we use small ones for secondary link out content (Tutorials). Should they or shouldn't they coexist on the same page? I think we just need to outline some guidelines one when we use them and how we use them.

----

**Step 2 (Access design tools)** 

<img width="818" alt="Screen Shot 2022-09-19 at 4 17 31 PM" src="https://user-images.githubusercontent.com/43969356/191108663-05d4d15f-3799-4c49-8347-0ec8c037fb46.png">

**Step 3 (tutorials)** 

<img width="781" alt="Screen Shot 2022-09-19 at 4 17 42 PM" src="https://user-images.githubusercontent.com/43969356/191108688-6e3dc60f-2333-4fed-85a0-31ddd668ad45.png">

**Reply by kingtraceyj** [reply]
#1326

**Comment by kingtraceyj** [comment]
📝 Fix - Filter scroll on mobile
**Page:** [Functions catalog](https://next.carbondesignsystem.com/assets/functions?q=)
**Issue:** The filter dropdown has an unexpected scroll when open. I expect there to be no scroll or very little scroll. I am unsure of what is actually scrolling.
![function scroll](https://user-images.githubusercontent.com/20582179/191110059-b1c200ca-20a1-4109-aaed-b0547676fa06.gif)

**Reply by alisonjoseph** [reply]
https://github.com/carbon-design-system/carbon-platform/issues/1322

**Comment by laurenmrice** [comment]
🎨 Visual
**Page:** https://next.carbondesignsystem.com/guidelines/content/overview#content-foundations
**Issue:** Italics versus regular text for links in structured lists?
**Suggestion:** We should align on if we use italics for links in structured lists. Sometimes we use italics and sometimes we use regular plex. 

----

Example of table with italic links on [Content foundations](https://next.carbondesignsystem.com/guidelines/content/overview#content-foundations)

<img width="792" alt="Screen Shot 2022-09-19 at 4 24 10 PM" src="https://user-images.githubusercontent.com/43969356/191109749-9c5df358-227e-4b01-9b83-0f622db0d567.png">

Example of table with no italic links on [Step 4: Review foundational guidance](https://next.carbondesignsystem.com/designing/get-started#step-4:-review-foundational-guidance)

<img width="794" alt="Screen Shot 2022-09-19 at 4 27 16 PM" src="https://user-images.githubusercontent.com/43969356/191110304-e3e44e50-6f8a-43c8-a761-7607591daef7.png">

**Reply by kingtraceyj** [reply]
#1327

**Comment by thyhmdo** [comment]
🧭 Navigation and IA feedback
**Page:** https://next.carbondesignsystem.com/design-kits
**Issue:** I'm missing the general context what I can search here. We now have design kits info at two different places which are disjointed and confusing. Under the "Designing" tab, there is info specific to each tool. People may want to read those first before searching or downloading kits. 
**Suggestion:** Maybe for first time users, it's good to point them to those pages under "Designing" with a general short description in Catalogs - Design kits. / Or indicate somehow if it's available for internal/external users. 

![image](https://user-images.githubusercontent.com/77697351/191110284-30d8396a-85a2-4262-b688-87671c09a8a5.png)
![image](https://user-images.githubusercontent.com/77697351/191110327-d70a9e51-0ce4-4871-b64e-4e69d9bc21f8.png)

**Reply by kingtraceyj** [reply]
#1328
